src/pages/Movies/Movies.js

```diff
@@ -1,78 +1,101 @@
 import React, { useEffect } from 'react';
 import { useState } from 'react';
+import { BASE_URL } from '../../config';
 import SelectOption from '../../components/SelectOption/SelectOption';
 import CardMovie from '../../components/Movie/Movie';
-import Detail from '../../components/Detail/Detail';
+// import Detail from '../../components/Detail/Detail';
+// import DetaiModal from '../Detailmodal/detaiModal';
 import styled from 'styled-components';
-import Common from '../../components/Common/Common';
 
 const Movies = () => {
-  const { IcoMovie } = Common;
   const [movieList, setMovieList] = useState([]);
-  const [isChecked, setIsChecked] = useState(false);
   const [isOpenModal, setIsOpenModal] = useState(false);
+  const [optionValue, setOptionValue] = useState('reservationRate');
+  const [onScreen, setOnScreen] = useState('2');
 
-  const handelCheck = e => {
-    setIsChecked(e.target.checked);
+  const handelOnScreen = e => {
+    setOnScreen(e.target.value);
   };
 
   const handleModal = () => {
     setIsOpenModal(prev => !prev);
   };
 
+  const filterAPI = `${BASE_URL}/movies?movieStatusesId=${onScreen}&filter=${optionValue}`;
+
   useEffect(() => {
-    fetch('https://jsonplaceholder.typicode.com/albums', {
+    fetch(filterAPI, {
       method: 'GET',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8', //필수로 넣어야함
+      },
     })
       .then(res => res.json())
       .then(data => {
-        setMovieList(data);
+        setMovieList(data.movieList);
       });
-  }, []);
+  }, [filterAPI]);
 
   return (
     <>
       <SubContents>
         <WrapTitle>
           <TitGlobal>무비차트</TitGlobal>
           <GroupTab role="tablist">
-            <BtnMovies type="button" role="tab" aria-selected="true">
+            <BtnMovies
+              type="button"
+              role="tab"
+              aria-selected={onScreen === '2' ? 'true' : 'false'}
```

Comment:
```suggestion
              aria-selected={onScreen === '2'}
```

조건에 대한 반환값은 boolean 값이라서 이렇게 해줘도 되는데
한가지 궁금한 점은 반드시 true 혹은 false 값이 string으로 들어가야 하는 부분이라면
받는 부분에서 string으로 처리해주세요!